pkg/driver/mount.go

```diff
@@ -19,44 +19,50 @@ package driver
 import (
 	"os"
 
-	"k8s.io/utils/exec"
-	"k8s.io/utils/mount"
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/mounter"
+	mountutils "k8s.io/mount-utils"
+	utilexec "k8s.io/utils/exec"
 )
 
-// Mounter is an interface for mount operations
 type Mounter interface {
-	mount.Interface
-	exec.Interface
+	// Implemented by NodeMounter.SafeFormatAndMount
+	mountutils.Interface
 	FormatAndMount(source string, target string, fstype string, options []string) error
-	GetDeviceName(mountPath string) (string, int, error)
-	MakeFile(pathname string) error
-	MakeDir(pathname string) error
-	ExistsPath(filename string) (bool, error)
+
+	// Implemented by NodeMounter.SafeFormatAndMount.Exec
+	// TODO this won't make sense on Windows with csi-proxy
```

Comment:
What are the TODOs here? Remove them?

Reply:
i don't know yet what if anything needs to be done on windows until i actually test it which could take a while so I droped a lot of these windows-related TODOS throughout the code.

Reply:
so far I've only tested up to the point where it gets the device path (device number on windows) . All of the subsequent execs and mkdirs and mkfiles, i expect to fail, but exactly how to fix that I don't kno